src/main/resources/application.properties

```diff
@@ -25,7 +25,7 @@ datarepo.resourceId=broad-jade-dev
 datarepo.userEmail=jhert.dev@gmail.com
 datarepo.dnsname=some.dnsname.org
 datarepo.userId="999999999999"
-datarepo.maxStairwayThreads=10
+datarepo.maxStairwayThreads=20
```

Comment:
Glad to see this increased. Any particular reasoning behind the number you chose?

Also, is this the number of threads being used in integration tests run by GH action also or will that require a separate change to one of the DR helm repos?

Reply:
I test the bulk load with a width of 10, so wanted to not use up all of the threads.
We need to do more perf work to understand how many threads we should have and how to budget them between different types of tasks.